### PR TITLE
Issue number- 245: updated docs/readme, svc-aggregation/readme, svc-systems/readme, svc-update/readme files

### PR DESCRIPTION
**Pull request by chinmayic**
svc-aggregation/ readme changes:
1. Added connection methods in the aggregation service root response.
2. Included connection method info in the request body for adding a plugin and BMC.
3. Removed OEM info from the request body for adding a plugin and BMC.

svc-update/readme changes:
Included Task responses for simple and start update actions.

svc-systems/readme changes:
1. Added single network adapter information.
2. Added PCIedevices information.
3. Modified response of storage subsystem for storage controllers.
4. Included the following note for creation and deletion of volumes:
    "Reset is required for the changes to reflect."

docs/readme changes:
1. All of the above changes.
2. Updated TOC.
3. Updated list of APIs.

### CODE REVIEW

File: svc-aggregation/README.md [hunk 1]
```diff
@@ -210,8 +331,9 @@ curl -i POST \
 |PluginID|String \(required\)<br> |The id of the plugin you want to add. Example: GRF \(Generic Redfish Plugin\), ILO<br> |
```
Comment (irappa-pattar) [inline svc-aggregation/README.md]: remove PluginID, PreferredAuthType and PluginType from the Request parameters

File: svc-aggregation/README.md [hunk 2]
```diff
@@ -323,42 +452,45 @@ curl -i -X POST \
...
+**Request parameters**
 
 |Parameter|Type|Description|
 |---------|----|-----------|
 |HostName|String \(required\)<br> |A valid IP address or hostname of a baseboard management controller \(BMC\).|
 |UserName|String \(required\)<br> |The username of the BMC administrator account.|
 |Password|String \(required\)<br> |The password of the BMC administrator account.|
 |Links \{|Object \(required\)<br> |Links to other resources that are related to this resource.|
-|Oem\{ PluginID \} \} |String \(required\)<br> |The plugin Id of the plugin.<br> NOTE: Before specifying the plugin Id, ensure that the installed plugin is added in the resource inventory. To know how to add a plugin, see [Adding a Plugin](GUID-4E64426F-559C-430A-AE60-61409DFB4131.md).| 
+|Oem\{ PluginID \} \} |String \(required\)<br> |The plugin Id of the plugin.<br> NOTE: Before specifying the plugin Id, ensure that the installed plugin is added in the resource inventory. To know how to add a plugin, see [Adding a Plugin](#adding-a-plugin-as-an-aggregation-source).|
```
Comment (irappa-pattar) [inline svc-aggregation/README.md]: remove Oem block from the Request Parameters

File: svc-aggregation/README.md [hunk 3]
```diff
@@ -488,7 +621,7 @@ curl -i GET \
 
 ```
 
-> Sample response body
+>**Sample response body**
 
 ```
 {
```
Comment (irappa-pattar) [inline svc-aggregation/README.md:623]: should be used connection method, not oem

**Review by irappa-pattar — CHANGES_REQUESTED** [review]
address the comments

File: svc-aggregation/README.md (hunk 2 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Removed.

File: svc-aggregation/README.md (hunk 1 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Removed

File: svc-aggregation/README.md (hunk 3 above)
Comment (chinmayic) [reply svc-aggregation/README.md:623]: Changed it.

**Review by irappa-pattar — APPROVED** [review]

**Review by tamanna-k — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**